mcs/class/corlib/Test/System/MathTest.cs

```diff
@@ -499,15 +499,15 @@ public void TestPow ()
 			Assert.IsTrue (double.IsNaN (Math.Pow (             double.NaN, double.NegativeInfinity)), "#2");
 			Assert.IsTrue (double.IsNaN (Math.Pow (             double.NaN,                      -2)), "#2");
 			Assert.IsTrue (double.IsNaN (Math.Pow (             double.NaN,                      -1)), "#3");
-			Assert.IsTrue (double.IsNaN (Math.Pow (             double.NaN,                       0)), "#4");
+			Assert.IsFalse (double.IsNaN (Math.Pow (             double.NaN,                       0)), "#4");
```

Comment:
Doesn't this break on top of .NET Framework?

Reply:
it does, but matches .NET Core and the IEEE 754 spec, see https://github.com/dotnet/coreclr/issues/9807

Reply:
or did you mean the case where we could run our tests on top of .NET Framework on Windows? AFAIR nobody maintains it and it doesn't work.